### DISCUSSION TITLE
Editing a transactions destination account doesnt save all though submit is successful.

### DISCUSSION THREAD

**Discussion by ahmaddxb** [victorbalssa/abacus#172] [Bug]
**Describe the bug**
Editing a transactions destination account doesnt save all though submit is successful.

My default currency is AED

**Expected behavior**
Editing a transaction destination account should update the transaction recored.

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: Samsung S22
 - OS: Android 13
 - Version 0.9.2

**Comment by victorbalssa** [comment]
@ahmaddxb You must select an Expense account as a `destination` when updating a Tx.
